doc/reference_accessor.xml

```diff
@@ -331,9 +723,9 @@ ST_Dimension
 	  <refsection>
 		<title>Description</title>
 
-		<para>Returns the last point of a <varname>LINESTRING</varname> geometry
-		as a <varname>POINT</varname> or <varname>NULL</varname> if the input
-		parameter is not a <varname>LINESTRING</varname>.</para>
+		<para>Returns the last point of a LINESTRING
+		as a POINT.  Returns <varname>NULL</varname> if the input
```

Comment:
Why is POINT removed as varname but LINESTRING isn't?
In any case they're not variables so semantically it's incorrect to mark them this way. Perhaps find some part of the doc that mentions geometry types and link there?

Reply:
Fixing this.

And good point about reason for not specifying geometry type names as varnames.  (I think the goal was just to get them in a code font, but either this should be done with a different tag, or just not done at all.  This is something for the style guide.